WcaOnRails/app/views/competition_tabs/_competition_tab_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if @competition_tab.new_record?
```

Comment:
This could go there but I'm not sure whether that's better. This is necessary to abstract this form and allow both new and edit templates to use it. The other way is to pass the two variables below as arguments to the calls of render (at least better than messing up the controller), what do you consider to be the most appropriate here?